### DISCUSSION TITLE
Update to QFieldSync v4.0.0

### DISCUSSION THREAD

**Discussion by Thomas-2022** [opengisch/QField#2814] [Q&A]
Ich habe eine Frage zum Update "v4.0.0": 
Da ich Projekte aktuell nicht mehr auf einfachem Weg für QField verpacken kann (da kommen immer seit neuestem Python-Fehler) wollte ich mir wie hier empfohlen, das genannte Update hinzuladen.
Wenn ich es dann hinzulade bei "Aus ZIP installieren" kommt zwar die Meldung "erfolgreich installiert", aber letztlich wurde es doch nicht installiert....

Google translate:
> I have a question about update "v4.0.0":
> Since I am currently no longer able to package projects for QField in a simple way (python errors have been coming since the latest), I wanted to download the update mentioned here, as recommended here.
> When I then add it to "Install from ZIP" I get the message "installed successfully", but ultimately it wasn't installed....

**Comment by suricactus** [comment]
@Thomas-2022 Please write in English.

Also, please provide more information about your issue. What are these "Python-Fehler"? Do you still experience these errors in 4.1.0?

**Reply by Thomas-2022** [reply]
I can't find the Update "4.1.0". 
There's only the Update "v4.0.0"...

**Reply by suricactus** [reply]
Which QGIS version are you using?

**Reply by Thomas-2022** [reply]
Sorry für das sehr späte Antworten....: 
Ich verwende QGIS 3.22.
Python-Fehler, zB: "...AppData/Roaming/QGIS/QGIS3\profiles\default/python/ plugins\qfieldsync\qfield_sync.py", line 381, in show_package_dialog self.push_dlg = PackageDialog(
Oder zB:
"...AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\qfieldsync\gui\package_dialog.py", line 78, in__init__ self.setup_gui()
